doc/user/command-line.xml

```diff
@@ -26,11 +26,10 @@ SPDX-License-Identifier: MIT
          xsi:schemaLocation="http://www.scons.org/dbxsd/v1.0 http://www.scons.org/dbxsd/v1.0/scons.xsd">
 <title>Controlling a Build From the Command Line</title>
 
-<para>
+  <para>
 
   &SCons; provides a number of ways
-  for you as the writer of the &SConscript; files
-  to give you (and your users)
+  for you as the build system writer to grant
```

Comment:
"build system engineer" instead of "build system writer" ?

Reply:
I find that almost as awkward, couldn't come up with a good term here. Will think a bit more.

Reply:
Kind of reworded the paragraph, I think it reads less awkward now.